examples/project-management-with-calendar/project-management-with-calendar.md

```diff
@@ -1,10 +1,10 @@
-# Personal Project Management with Google Calendar API
+# Project management with Google Calendar API
 
 Let's explore how Alex, a software developer, leverages the Google Calendar API in Ballerina for managing his personal project schedule and collaborating with team members.
 
 ## Step 1: Import Google Calendar Connector
 
-Alex begins by importing the `ballerinax/googleapis.gcalendar` module into his Ballerina project.
+Alex begins by importing the `ballerinax/googleapis.gcalendar` package.

```

Comment:
Module

Reply:
Fixed in a new pr https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/149/